code/modules/paperwork/records/info/waw.dm

```diff
@@ -150,55 +152,55 @@
 	Name : General Bee	<br>
-	Risk Class : Waw	<br>
+	Risk Class : WAW	<br>
 	Max PE Boxes : 24	<br>
 	Qliphoth Counter : 1	<br>
 	Work Damage Type : Red	<br>
 	Work Damage : High	<br>
 	- When the work result was Normal, the Qliphoth Counter lowered with a normal probability.	<br>
 	- When the work result was Bad, the Qliphoth Counter lowered with a high probability.	<br>
-	- When the Qliphoth Counter reached 0, General Bee dispatched a squad of soldiers to the facility..	<br>
-	WARNING - Do not let Queen Bee spores come in contact with General Bee.	<br>
+	- When the Qliphoth Counter reached 0, General Bee dispatched a squad of soldiers to the facility.	<br>
+	- WARNING: Do not let Queen Bee spores come in contact with General Bee.	<br>
 	<br>
 	<h4>Instinct:</h4> High<br>
 	<h4>Insight:</h4> Common<br>
 	<h4>Attachment:</h4> Very Low<br>
 	<h4>Repression:</h4> Common<br>
 	<h4>Escape Damage Type:</h4> Red/Black	<br>
 	<h4>Escape Damage:</h4> High	<br>
-	<h4>Red Resistance:</h4> Endured	<br>
-	<h4>White Resistance:</h4> Ineffective	<br>
-	<h4>Black Resistance:</h4> Ineffective	<br>
+	<h4>Red Resistance:</h4> Resistant	<br>
+	<h4>White Resistance:</h4> Endured	<br>
+	<h4>Black Resistance:</h4> Endured	<br>
 	<h4>Pale Resistance:</h4> Normal	<br>"}
 
 //Shrimp
 /obj/item/paper/fluff/info/waw/shrimpexec
 	name = "O-02-119"
 	info = {"<h1><center>O-02-119</center></h1>	<br>
-	Name : A Shrimp Association Executive	<br>
-	Risk Class : Waw	<br>
+	Name : Shrimp Association Executive	<br>
+	Risk Class : WAW	<br>
 	Max PE Boxes : 24	<br>
 	Qliphoth Counter : 1	<br>
 	Work Damage Type : White	<br>
 	Work Damage : Moderate	<br>
 	- When the work result was Bad, the Qliphoth Counter lowered.	<br>
-	- When the Qliphoth Counter reached 0, the executive called for help.	<br>
-	- When the work result was Good, the executive ordered a gift for NAME.	<br>
-	- After work completed, A Shrimp Association Executive requested something from NAME	<br>
-	- Doing the requested work with A Shrimp Association Executive increased success rate significantly. All materials were kept in it's containment, NAME was just required to do the corresponding work.	<br>
-	- The gifts included : <br>
-	Various Shrimp Corporation brand firearms.	<br>
-	A "Shrimp in a grenade"<br>
-	A can of Wellcheers-Brand soda<br>
-	<br>
+	- When the Qliphoth Counter reached 0, a shrimp strike force arrived at Shrimp Association Executive’s location.	<br>
+	- After an employee completed their work, Shrimp Association Executive requested an idea or service. 	<br>
+	- Observations have concluded that each request corresponded to a specific work type. Performing this work type will result in significantly increased work success rate. 	<br>
+	- When the work result was Good, Shrimp Association Executive ordered shrimp-themed merchandise for the employee. Possible items included:
+	<ol type="1">
+		<li>Shrimp Corporation branded firearms</li>
+		<li>"shrimp in a grenade"</li>
+		<li>Wellcheers soda</li>
```

Comment:
"A can of Wellcheers-Brand soda" fits better here, as it does not assume that the reader knows what wellcheers is